src/components/tables/torrenttable.tsx

```diff
@@ -299,13 +299,24 @@ function ByteRateField(props: TableFieldProps) {
 }
 
 function PercentBarField(props: TableFieldProps) {
-    const now = props.torrent[props.fieldName] * 100;
+    let now: number = props.torrent[props.fieldName] * 100;
+    let label: string = '';
+    let status: number = props.torrent.status;
+    if ((props.torrent.error !== undefined && props.torrent.error > 0) || props.torrent.cachedError !== "") {
+        status = Status.error;
+    } else if (props.torrent.status === Status.downloading && props.torrent.pieceCount === 0) {
+        now = 100;
+        label = "🧲";
```

Comment:
No emojis and no custom labels here, the icon in name field is enough indication.